apps/showcase/scripts/prepare-training-exercises/index.cjs

```diff
@@ -0,0 +1,27 @@
+const {readdir, readFile, writeFile} = require('node:fs/promises');
+const {getFilesTree} = require('@o3r/training-tools');
+const {dirname, join, resolve} = require('node:path');
+const glob = require('globby');
+
+/**
+ * The Chrome Extension mechanism does not accept exports in loaded module
+ * The purpose of this script is to remove the empty `export` generated by TSC
+ */
+
+void (async () => {
+  const root = resolve(__dirname, '..', '..');
+  const files = await glob('src/assets/*-training/**/(exercise|solution)/**', {cwd: root});
```

Comment:
[Personal preferences] Couldn't we have the following folder structure `src/assets/trainings/<topic>/...` ?